plugins/azure-devops/src/hooks/usePullRequests.ts

```diff
@@ -36,15 +36,18 @@ export function usePullRequests(
   loading: boolean;
   error?: Error;
 } {
-  const top = defaultLimit ?? AZURE_DEVOPS_DEFAULT_TOP;
+  const api = useApi(azureDevOpsApiRef);
+  const config = useApi(configApiRef);
+  const configDefaultLimit = config.getOptionalNumber(
+    'azureDevOps.repos.defaultLimit',
+  );
+  const top = defaultLimit ?? configDefaultLimit;
```

Comment:
It might be clearer how this behaves if the value defaults are done in one place

```suggestion
  const top = defaultLimit ?? config.getOptionalNumber('azureDevOps.repos.defaultLimit') ??  AZURE_DEVOPS_DEFAULT_TOP;
```

Reply:
This also applies to `plugins/azure-devops/src/hooks/useBuildRuns.ts` and polling interval in `plugins/azure-devops/src/hooks/useDashboardPullRequests.ts`